packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2701,10 +2701,31 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   ///
   /// Used by coordinate conversion functions to translate coordinates local to
   /// one render object into coordinates local to another render object.
+  ///
+  /// Some RenderObjects will provide a zeroed out matrix in this method,
+  /// indicating that even though
```

Comment:
indicating that even though ....?

Reply:
Hah. I got halfway through writing this and thought of what I should say in `paintsChild` and forgot to come back. Updated.